### DISCUSSION TITLE
Enable County field in the QTH profile

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#3132] [Ideas / Feature Requests]
Now it is possible to add field for state (Oblast in Russia) in station location QTH data. It would be good to make possibility to edit County field (RDA in Russia) there. And export this data into ADIF file,

![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/94406b83-943a-4889-8ad2-be72837e68f2)

**Comment by magicbug** [comment]
Other countys certainly can be supported (needs some additional code) what I'd need is a list formatted the same way as the US data set see 
 https://github.com/magicbug/Cloudlog/blob/master/assets/json/US_counties.csv

At the moment my focus is getting all the state data added lots more countries to add!

**Reply by R1BLH** [reply]
Great!
I'll attach the file in a few minutes. One question: how to handle RDA names in Russian language?

**Reply by magicbug** [reply]
Add it to the csv as an extra would be my suggestion

**Reply by R1BLH** [reply]
Here is it. 

[RDA.csv](https://github.com/user-attachments/files/15765005/RDA.csv)

**Reply by magicbug** [reply]
Thanks, will look at this next week

**Reply by R1BLH** [reply]
Thanks.
I have already done PR https://github.com/magicbug/Cloudlog/pull/3133 for the correct RDA. I think "if" should be changed to "switch" (will do it tonight). 
I filled MY_CNTY field in my log database with correct RDA data. Export works correctly.
